components/gc-minister/index.json-ld

```diff
@@ -164,6 +168,67 @@
 					}
 				]
 			}
+		},
+		{
+			"@id": "_:implement_minister_2",
+			"iteration": "_:iteration_minister_2",
+			"name": {
+				"en": "Standard",
+				"fr": "Standard"
+			},
+			"introduction": {
+				"en": "This implementation is meant for developers/publishers adding the component manually.",
+				"fr": "Cette implémentation est destinée aux développeurs/éditeurs qui ajoutent le composant manuellement."
+			},
+			"instructions": {
+				"en": [
+					"The following coded example is the default layout. When you are implementing your instance, you can omit any of the following:<ul><li>image</li><li>mandate letter link</li><li>briefing book link</li></ul>",
+					"Add the class <code>col-xs-12</code> to the <code>col-md-5</code> and <code>col-md-7</code> elements (containing the image and description respectively) in order to fix issue where links are not clickable in mobile view",
+					"The <code>&lt;div class=\"col-md-6\"></code> can be repeated as many times as there are ministers or institutional heads.",
+					"Refer to the official guidance for the appropriate section heading."
+				],
+				"fr": [
+					"L'exemple codé suivant est la présentation par défaut. Lorsque vous mettez en œuvre votre instance, vous pouvez omettre les éléments suivants&nbsp;:<ul><li>Image</li><li>Lettre de mandat</li><li>Dossier d’information</li></ul>",
+					"Ajoutez la classe col-xs-12 aux éléments col-md-5 et col-md-7 (contenant respectivement l'image et la description) afin de résoudre le problème où les liens ne sont pas cliquables dans la vue mobile",
+					"Le <code>&lt;div class=\"col-md-6\"></code> peut être répété autant de fois qu'il y a de ministres ou de chefs d'institution.",
+					"Reportez-vous aux directives officielles pour connaître le titre de section approprié."
+				]
+			},
+			"notes": {
```

Comment:
No need to duplicate these notes.

Reply:
Fixed in commit " Updated documentation"